content/docs/troubleshooting/README.md

```diff
@@ -1,8 +1,20 @@
 ---
 title: Troubleshooting
-description: 'cert-manager FAQ: Troubleshooting'
+description: |
+    Learn how to debug common problems with cert-manager
 ---
 
+In this section, you will learn troubleshooting techniques that will help you find the root cause if your Certificate fails to be issued or renewed.
```

Comment:
I can't quite explain why I want to make this change:
```suggestion
In this section, you will learn troubleshooting techniques that will help you find the root cause when your Certificate fails to be issued or renewed.
```

(I'm not attached to it--I'd be more attached if I could explain my rationale.)

The same change would apply in other instances, but only consider it/them if someone else can justify the change in general.

Reply:
We didn't want to imply that it would *inevitably* fail :-) I'll leave it for now.